packages/components/board/stories/board.stories.js

```diff
@@ -0,0 +1,16 @@
+import { storiesOf, html } from '@open-wc/demoing-storybook';
+import { withKnobs, text } from '@storybook/addon-knobs';
+
+import '@tradeshift/elements.board';
+
+storiesOf('ts-board', module)
+	.addDecorator(withKnobs)
+	.add('default', () => {
+		const title = text('My Board', 'Title');
+
+		return html`
+			<ts-board data-title="${title}">
+				This my first board.
```

Comment:
Can you please make this part also dynamic so people can provide `html` in the slot?